### DISCUSSION TITLE
Jetbrains student pack Blocked mail address

### DISCUSSION THREAD

**Discussion by TolunayOzcan** [community/community#67937] [GitHub Education]
### Hi everyone,

I'm making this mistake, what should I do?
![IMG_2584](https://github.com/community/community/assets/145322256/a2a8d35d-fe95-4cc8-a851-5d11a22cec7c)

**Comment by seenv** [comment]
Based on the info provided on [here](https://intellij-support.jetbrains.com/hc/en-us/community/posts/6039589923218-student-license-application-email-blocked) and [here](https://intellij-support.jetbrains.com/hc/en-us/community/posts/9742579083282-Jetbrains-student-access-being-blocked), you should apply through "Official Documents".
